.gitignore

```diff
@@ -4,3 +4,4 @@ xcuserdata
 *.xcodeproj
 DerivedData/
 .DS_Store
+Package.resolved
```

Comment:
This shouldn't be checked in

Reply:
Really? It is also part of e.g. Sugar's .gitignore.

Reply:
Hah my bad - was a bit too quick there. My brain somehow thought that you were checking in that file (and not ignoring it) 😄